Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2
# KINDA Vanilla Hairs

## Changelog:
- Made a variation of Straight Pointy Beard called Straight Pointy Goatee for the males.
- Added a new male hairstyle, Bald Top.


# How to test:
Open the Ruler Designer, find the new hair and beard. tell me your opinion.

Alternatively, to take a closer look at Bald Top:
1) Run the game with the [debug mode](https://ck3.paradoxwikis.com/Console_commands#Enabling_debug_mode)
2) Press `~`, the tilde, click **_Portrait Editor_**
3) In the ethnicity list find `green_orc`, click **_Randomize DNA_**.
4) Click **_3D View_** and move the opened window somewhere to empty space. You can rotate the character there.
5) Spam **_Randomize DNA_** until you see the bald top hairstyle.
